configs/projects/_shared-agent-components.rb

```diff
@@ -48,8 +48,8 @@
 proj.component 'rubygem-gettext'
 proj.component 'rubygem-fast_gettext'
 
+proj.component 'rubygem-ffi'
```

Comment:
@sebastian-miclea this should not be build on all platforms on 5.5.x and 6.4.x because we will not use facter-ng on those streams